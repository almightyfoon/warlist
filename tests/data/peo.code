ce501b_-0rhth2h2hyhyh5h3h3hlhmhmhmhnh9hr___-0shvh2h4h4j6hlhoh9hbhjhbhjhehehh
