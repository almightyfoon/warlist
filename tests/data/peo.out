https://conflictchamber.com/#ce501b_-0rhth2h2hyhyh5h3h3hlhmhmhmhnh9hr___-0shvh2h4h4j6hlhoh9hbhjhbhjhehehh

Grymkin Army - 75 / 75 points
[Theme] Dark Menagerie

[Dreamer 1] The Dreamer [+28]
 - Cage Rager [14]
 - Cage Rager [14]
 - Crabbit [0(-)]
 - Crabbit [0(-)]
 - Frightmare [9]
 - Skin & Moans [15]
 - Skin & Moans [15]
Glimmer Imp [4]
Gremlin Swarm [0(3)]
Gremlin Swarm [0(3)]
Gremlin Swarm [3]
Lady Karianna Rose [4]
Dread Rots (max) [12]
Death Knell [13]

Grymkin Army - 75 / 75 points
[Theme] Bump in the Night

[King of Nothing 1] The King of Nothing [+28]
 - Cage Rager [14]
 - Crabbits (2) [7]
 - Crabbits (2) [7]
Eilish Garrity, the Occultist [0(5)]
Glimmer Imp [0(4)]
Lord Longfellow [0(6)]
Dread Rots (max) [12]
Hollowmen (max) [13]
 - Lantern Man [3]
Hollowmen (max) [13]
 - Lantern Man [3]
Neigh Slayers (max) [12]
Neigh Slayers (max) [12]
Twilight Sisters [7]
